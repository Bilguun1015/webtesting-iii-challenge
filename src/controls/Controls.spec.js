// Test away
import React from "react";
import renderer from "react-test-renderer"; 
import { render, fireEvent } from "@testing-library/react";

import Controls from './Controls';

describe('<Controls />', () => {
    it('matches snapshot', () => {
        const tree = renderer.create(<Controls />);

        expect(tree.toJSON).toMatchSnapshot();
    });
    it('should display a message Lock Gate', () => {
        
        const { getByText, queryByText } = render(<Controls />)
        fireEvent.click(getByText('Close Gate'))
        expect(queryByText(/Open Gate/i)).toBeFalsy();
    })
})