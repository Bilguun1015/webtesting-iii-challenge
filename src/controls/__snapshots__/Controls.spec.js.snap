// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Controls /> matches snapshot 1`] = `[Function]`;
